.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: run-acceptance-test
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    pipelinesascode.tekton.dev/pipeline: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/pipelines/acceptance-tests.yaml"
+    pipelinesascode.tekton.dev/on-target-branch: "[master]"
```

Comment:
I'm OK to start with master but IMHO it would make sense also on release branches. Is it possible to use regexp here?